uobtheatre/site_messages/models.py

```diff
@@ -105,7 +106,7 @@ class Policy(models.TextChoices):
         max_length=7,
         choices=Policy.choices,
         default=Policy.DEFAULT,
-        help_text="The policy for a message's dismissal. By default messages are dismissable, and this choice is stored in the cache until the event is over. Single-Session Only messages can be dismissed, but dismissal is not cached. Prevented messages cannot be dismissed by the user.",
+        help_text="The policy for a message's dismissal. By default messages are dismissable, and this choice is stored in the cache until the event is over. Single-Session Only messages can be dismissed, but dismissal is not cached. Prevented messages cannot be dismissed by the user: if used for a modal, this will essentially shut down that function (or the whole site, for a sitewide modal).",
```

Comment:
I understand the difference now between `Single` and `Default` settings for dismissal, thanks to your [PR comment](https://github.com/BristolSTA/uobtheatre-web/pull/992#discussion_r2395649044). I wonder though if that's still entirely clear here with the `cached`/`not cached` language, or if we should just say: `By default messages are dismissible, and once a message has been dismissed by a user, it will never reappear for them. Single-Session Only messages can be dismissed, but will reappear whenever the page is loaded.`, for example.